exercises/diffie-hellman/src/test/kotlin/DiffieHellmanTest.kt

```diff
@@ -0,0 +1,79 @@
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Test
+import java.math.BigInteger
+import kotlin.test.assertEquals
+import kotlin.test.assertNotEquals
+import kotlin.test.assertTrue
+
+class DiffieHellmanTest {
+
+    private lateinit var diffieHellman: DiffieHellman
+
+    @Before
+    fun setUp() {
+        diffieHellman = DiffieHellman()
```

Comment:
After you reimplemented `DiffieHellman` to `object`, the test should accomodate new signature, as `DiffieHellman()` can no longer be treated as ctor.